Sources/AuthFoundation/Network/Internal/String+AuthFoundation.swift

```diff
@@ -54,7 +54,7 @@ private let systemVersion: String = {
     #endif
 }()
 
-public class SDKVersion {
+public class SDKVersion: Codable {
```

Comment:
Will revert this; Codable was to be used when tracking which versions had been migrated, but that seems to be unnecessary at this point.